src/environmentd/tests/auth.rs

```diff
@@ -810,8 +810,10 @@ fn test_auth_expiry() {
     let encoding_key =
         EncodingKey::from_rsa_pem(&ca.pkey.private_key_to_pem_pkcs8().unwrap()).unwrap();
 
-    const EXPIRES_IN_SECS: u64 = 40;
-    const REFRESH_BEFORE_SECS: u64 = 20;
+    const EXPIRES_IN_SECS: u64 = 20;
```

Comment:
In theory we can keep increasing this number to reduce flakiness without affecting test time.

Reply:
In fact, increasing this to `100000000` locally does not affect the test time.